src/services/tokenService.js

```diff
@@ -0,0 +1,28 @@
+const Token = require('../models/token.js');
+
+const save = async (userId, newToken) => {
+  const token = await Token.findOne({ where: { userId } });
```

Comment:
Consider adding error handling for the database operation to catch and log any errors that occur during the token retrieval process. This will help in diagnosing issues if the operation fails.